auraed/src/lib.rs

```diff
@@ -237,25 +237,32 @@ impl AuraedRuntime {
         // Initialize the bundler
 
         // Build gRPC Services
+        let (mut health_reporter, health_service) =
+            tonic_health::server::health_reporter();
+
         let cell_service = CellService::new();
         let cell_service_server = CellServiceServer::new(cell_service.clone());
+        health_reporter.set_serving::<CellServiceServer<CellService>>().await;
 
         let discovery_service = DiscoveryService::new();
         let discovery_service_server =
             DiscoveryServiceServer::new(discovery_service.clone());
+        health_reporter.set_serving::<DiscoveryServiceServer<DiscoveryService>>().await;
 
         let pod_service = PodService::new(self.runtime_dir.clone());
         let pod_service_server = PodServiceServer::new(pod_service.clone());
+        health_reporter.set_serving::<PodServiceServer<PodService>>().await;
```

Comment:
should this come after we've spawned the server below?  it likely doesn't matter but...

Reply:
I get that, but you can't actually read from the health reporter, so I think it's okay to leave it here and keep each services setup grouped.